Functions/TestDrive.Tests.ps1

```diff
@@ -1,6 +1,13 @@
 Set-StrictMode -Version Latest
 
-$tempPath = (Get-Item $env:temp).FullName
+if ($PSVersionTable.PSVersion.Major -lt 6 -or $IsWindows)
```

Comment:
You are repeating the implementation of Get-TempDirectory here. Use the function directly instead.